### DISCUSSION TITLE
GitHub Nasıl Puan Alıcaz

### DISCUSSION THREAD

**Discussion by komerli** [IBAX-io/go-ibax#561] [Bounty Hunter]
GitHub Nasıl Puan Alıcaz

**Comment by umit65** [comment]
once projeyi inceleyin derim

**Comment by Filiz23** [comment]
Biz inceledik ama nasıl puan alacağımızı bualamadık

**Comment by koray233** [comment]
Çok iyi proje

**Comment by umit65** [comment]
Telegramdan yaz

24 Oca 2022 Pzt 21:07 tarihinde koray233 ***@***.***> şunu
yazdı:

**Comment by umit65** [comment]
Telegramdan yaz

24 Oca 2022 Pzt 20:53 tarihinde Filiz23 ***@***.***> şunu
yazdı:
